source/includes/common-mc-admin-config.rst

```diff
@@ -1156,109 +1156,104 @@ Enabling or disabling data compression does not change existing objects.
 
 Comma-separated list of the file extensions to compress.
 Setting a new list of file extensions replaces the previously configured list.
-Defaults to ``.txt,.log,.csv,.json,.tar,.xml,.bin``.
+Defaults to ``".txt, .log, .csv, .json, .tar, .xml, .bin"``.
```

Comment:
Since we don't trim it, we probably shouldn't have spaces here.

Reply:
I tested it with spaces and it worked for me? Some hidden magic going on there? @klauspost

Reply:
Per discussion, keeping the spaces since my testing showed them to work. If there are hidden quirks, can address later.